script.js

```diff
@@ -167,7 +190,7 @@
 
         if ( requestPage && requestPage.length ) {
             var $entry = jQuery('<li/>').addClass('requestPage').text(requestPage).appendTo($appendTo);
-            guiElementActions(actions, requestPage, $entry);
+            guiElementActions([ORPHANACTIONS.view('Page')], requestPage, $entry);
```

Comment:
Hm, I think this will also remove the ability to open media files and the delete option for orphaned pages.